docs/conf/prague/2018/sponsors/prospectus.rst

```diff
@@ -182,11 +182,11 @@ Our job fair is a wonderful place to connect with the talented Write the Docs at
 We'll have over {{about.attendees}} people in attendance for our conference,
 and many of those people will come to our job fair to look for new roles and positions.
 
-**{{sponsorship.job_fair.price}}** (a la carte price, also included or discounted with other sponsorship packages)
+It is included in all sponsorship packages this year.
```

Comment:
even in first draft? I thought that it's just from second draft and upwards...

Reply:
I think the more the merrier -- I doubt the first draft folks will use it, but no real reason not to.